services/predictors/layoutparser/integration_tests/test_invocations.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+import requests
+from unittest import TestCase
+from mmda.types.image import Image
+from model.prediction import Prediction, BoxGroup, Box
+import os.path
+
+HOST = str(
+    subprocess.run(
+        "/sbin/ip route|awk '/default/ { print $3 }'", shell=True, capture_output=True
+    ).stdout.strip(),
+    "ascii",
+)
+PORT = 8080
+URL = f"http://{HOST}:{PORT}/invocations"
+
+
+def get_test_request():
+    return {"instances": [{}]}
+
+
+def get_expected_response():
+    return dict(
+        predictions=[
+            dict(output_field="asdf:4.2"),
+            dict(output_field="fdsa:6.4"),
+            dict(output_field="qwer:-2.0"),
+        ]
+    )
+
+
+class TestInvocations(TestCase):
+    def test__invoke(self):
+        page_images = [
```

Comment:
End-to-end integration test. This runs in reasonable time on a CPU. (Less time than it takes to build the image, anyway)